### ISSUE TITLE
Passages with multiple answer spans

### ISSUE THREAD

**Issue by TZeng20** [facebookresearch/DPR#233]
Hi,
I'm trying to implement the reader model training in Tensorflow.  How does DPR deal with multiple answer spans in a positive passage?
Will there just be multiple positions labelled as the correct start/end position?

E.g. if there are 2 answer starts in a passage (`h_i`, `h_j`), would the model try to maximise the scores for tokens `i` and `j` using cross entropy loss?
